templates/default/solr-sync.sh.erb

```diff
@@ -1,7 +1,7 @@
 #!/bin/sh
 
 CORE_NAME="<%= node['datashades']['app_id'] %>-<%= node['datashades']['version'] %>"
-LOCAL_DIR="/tmp"
+LOCAL_DIR="/mnt/local_data/solr_backup"
```

Comment:
should we not be checking if it exists and then use it, else use /tmp ?

Reply:
By the time 'setup' runs, the mount process should be well and truly complete. And every instance has the extra disk now.